hr_holidays_compute_days/models/hr_holidays.py

```diff
@@ -162,7 +162,8 @@ def _onchange_data_hr_holidays_compute_days(self):
                 include_rest_days=not self.holiday_status_id.exclude_rest_days,
                 compute_full_days=self.holiday_status_id.compute_full_days,
             )
-            days = employee.get_work_days_count(
-                from_datetime=date_from, to_datetime=date_to,
-            )
-            self.number_of_days_temp = days or 0
+            if employee:
```

Comment:
In the case of no employee is found, can we put a default value like 0 ?
Like set days equal to 0, then if there is an employee make some computation then in all case set number_of_days_temp = days 

What do you think about this approach?